### ISSUE TITLE
get_content_types() not working. throwing AttributeError

### ISSUE THREAD

**Issue by jeremyschiemann** [kiwicom/schemathesis#469]
I'm receiving this for nearly all endpoint in schemas.py in get_content_types()

```python
>   content_types = [list(definition.keys()) for definition in definitions]
E   AttributeError: 'str' object has no attribute 'keys'
```

probably has something to do that in `definitions = responses.get(str(response.status_code), {}).values()` .values() returns a dict_values object and not dict object

schemathesis 1.0.3, python 3.8.1

**Comment by Stranger6667** [comment]
Ah, I am very sorry. Will fix it right away - my test was wrong

**Comment by Stranger6667** [comment]
I forgot about `content` key

**Comment by Stranger6667** [comment]
Fixed in `1.0.4` - https://pypi.org/project/schemathesis/1.0.4/

Thank you very much for reporting it fast, very appreciated
